plugins/tiddlywiki/multiwikiserver/modules/mws-server.js

```diff
@@ -410,6 +410,14 @@ Server.prototype.requestAuthentication = function(response) {
 	}
 };
 
+// Check if the anonymous IO configuration is set to allow both reads and writes
+Server.prototype.isAnnonIOConfigured = function() {
+	const allowReadsTiddler = this.wiki.getTiddler("$:/config/MultiWikiServer/MultiWikiServerAllowAnnonymousReads");
```

Comment:
I think `wiki.getTiddlerText()` has a default value as a second parameter. So you should be able to define defaults using that function.